aten/src/ATen/quantized/Quantizer.cpp

```diff
@@ -487,16 +525,16 @@ inline Tensor new_qtensor_cpu(
   AT_ASSERT(options.device().is_cpu());
 
   native::check_size_nonnegative(sizes);
-  auto* allocator = at::getCPUAllocator();
+  static QNNPACKAllocator allocator;
```

Comment:
what about non-qnnpack environment?

Reply:
You mean FBGEMM uses this code as well?  In that case we shouldn't be using this allocator.  That is decided by a compile-time flag, right?

Reply:
this should probably be guarded by qengine and compiler flag as well if this is only compilable on qnnpack I think.